src/diffusers/pipelines/fabric/pipeline_fabric.py

```diff
@@ -0,0 +1,552 @@
+        neg_scale: float = 0.5,
+        pos_bottleneck_scale: float = 1.0,
+        neg_bottleneck_scale: float = 1.0,
+        output_type: Optional[str] = "pil",
+        latents: Optional[torch.FloatTensor] = None,
+    ):
+        r"""
+        Function invoked when calling the pipeline for generation. Generate a trajectory of images with binary
+        feedback. The feedback can be given as a list of liked and disliked images.
+
+        Args:
+            prompt (`str` or `List[str]`, *optional*):
+                The prompt or prompts to guide the image generation. If not defined, one has to pass `prompt_embeds`.
+                instead.
+            negative_prompt (`str` or `List[str]`, *optional*):
+                The prompt or prompts not to guide the image generation. If not defined, one has to pass
+                `negative_prompt_embeds` instead. Ignored when not using guidance (i.e., ignored if `guidance_scale` is
+                less than `1`).
+            liked (`List[Image.Image]` or `List[str]`, *optional*):
+                Liked enables feedback through images, encourages images with liked features.
+            disliked (`List[Image.Image]` or `List[str]`, *optional*):
+                Disliked enables feedback through images, discourages images with disliked features.
+            generator (`torch.Generator` or `List[torch.Generator]` or `int`, *optional*):
+                One or a list of [torch generator(s)](https://pytorch.org/docs/stable/generated/torch.Generator.html),
+                can be int. to make generation deterministic.
+            num_images (`int`, *optional*, defaults to 1):
+                The number of images to generate per prompt.
+            guidance_scale (`float`, *optional*, defaults to 7.5):
+                Guidance scale as defined in [Classifier-Free Diffusion Guidance](https://arxiv.org/abs/2207.12598).
+                `guidance_scale` is defined as `w` of equation 2. of [Imagen
+                Paper](https://arxiv.org/pdf/2205.11487.pdf). Guidance scale is enabled by setting `guidance_scale >
+                1`. Higher guidance scale encourages to generate images that are closely linked to the text `prompt`,
+                usually at the expense of lower image quality.
+            num_inference_steps (`int`, *optional*, defaults to 50):
+                The number of denoising steps. More denoising steps usually lead to a higher quality image at the
+                expense of slower inference.
+
+        Examples:
+            >>> from diffusers import FabricPipeline >>> import torch >>> model_id =
+            "dreamlike-art/dreamlike-photoreal-2.0" >>> pipe = FabricPipeline(model_id, torch_dtype = torch.float16)
+            >>> pipe = pipe.to("cuda") >>> prompt = "a giant standing in a fantasy landscape best quality" >>> liked =
+            [] >>> disliked = [] >>> image = pipe(prompt, num_images=4, liked=liked,disliked=disliked).images
```

Comment:
We can clear this and instead, we can add the example like so:

https://github.com/huggingface/diffusers/blob/74d902eb59f873b6156621220937f8e2521dfdc0/src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl.py#L556